functions/src/index.ts

```diff
@@ -24,13 +25,17 @@ import {
 } from "./utils";
 
 const firestoreDb = firebaseApp.firestore();
+firestoreDb.settings({ ignoreUndefinedProperties: true });
 const urlCollection = firestoreDb.collection(SHARE_LINK_FIRESTORE_COLLECTION);
 const apiKeyHandler = new APIKeyHandler(firestoreDb);
 const app = express();
 app.use(cors({ origin: "*" }));
+app.use(compression());
+// TODO: Add minInstances: 1 to prevent/limit cold starts once this is used in production.
+// See: https://firebase.google.com/docs/functions/manage-functions#min-max-instances 
 const runtimeOpts = {
   timeoutSeconds: 90,
-  memory: "1GB" as "1GB",
+  memory: "2GB" as "2GB",
```

Comment:
This feels a little wasteful, but I saw a decent bump in performance (1 to 1.5 seconds saved) with this increase

Reply:
>  think you can get rid of as "2GB" if you do const `runtimeOpts: functions.RuntimeOptions = { ..`

Ahh, nice--thank you! and good call, added a comment